Source/WebKit/NetworkProcess/webrtc/NetworkRTCProvider.h

```diff
@@ -157,8 +160,6 @@ class NetworkRTCProvider : private FunctionDispatcher, private IPC::MessageRecei
 
     void assertIsRTCNetworkThread();
 
-    Ref<NetworkRTCMonitor> protectedRTCMonitor();
```

Comment:
It does not seem protectedRTCMonitor is used elsewhere than in NetworkRTCProvider.

Reply:
Oh, right. This just got removed. Will revert that.